options/rtdl/generic/main.cpp

```diff
@@ -95,10 +95,17 @@ extern "C" [[ gnu::visibility("default") ]] void *__rtdl_allocateTcb() {
 	return allocateTcb();
 }
 
-extern "C" [[ gnu::visibility("default") ]] void _dl_debug_state()
-{
+DebugInterface debug;
+DebugInterface *_dl_debug_addr = &debug;
+
+extern "C" {
+	static void dl_debug_state() {
+
+	}
```

Comment:
Add a comment that GDB sets a breakpoint here but that we don't need to do anything.